releases/release-1.19/release-notes-draft.md

```diff
@@ -30,7 +30,7 @@
     plugins implement an `Unreserve` call.
   - `.disablePreemption` was removed. Users can disable preemption by disabling the
     "DefaultPreemption" PostFilter plugin. ([#91420](https://github.com/kubernetes/kubernetes/pull/91420), [@pancernik](https://github.com/pancernik)) [SIG Scheduling]
- 
+
```

Comment:
Do we wanna revert those spacing changes?

Reply:
done @saschagrunert